app/src/main/res/layout/contract_detail_your_info_row.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
Can this be a `LinearLayout`?

Reply:
Come to think of it, yes. We can probably use a mix of gravity and weights to represent it. I'm not sure that it will be more efficient, though 🤔

Reply:
I think I'm going to go with the `ConstraintLayout` here, because we would need to use the more heavy features of `LinearLayout` here to make that work.